CppCoreGuidelines.md

```diff
@@ -22295,7 +22295,7 @@ Never allow an error to be reported from a destructor, a resource deallocation f
 
         void test()
         {
-            std::array<Nefarious, 10> arr; // this line can std::terminate(!)
+            std::array<Nefarious, 10> arr; // this line can std::terminate()
```

Comment:
here exclamation point is not a typo, it is to emphasize the surprise. I suppose you could move it outside terminat's () to the end of the sentence.